src/app/(app)/course/[courseId]/students/(hooks)/useStudents.tsx

```diff
@@ -58,9 +58,9 @@ const createQueryString = (params: Record<string, string | number | undefined>,)
   })
   return searchParams.toString()
 }
-const FIXED_COURSE_NUM = 10
 
 const useStudentsProvider = (): StudentsContextType => {
+  const { courseId } = useParams<{ courseId: string }>()
```

Comment:
wait if we made useCourse, why are we not using it? Fetching it from the url params again forces you to revalidate, meaning repeating work that the CourseProvider is already doing